cypress/e2e/focus.cy.js

```diff
@@ -0,0 +1,57 @@
+const waitTime = 500;
+const viewports = { "desktop": 'macbook-13', "mobile": 'iphone-6+' };
+const domain = 'http://localhost:3000'
```

Comment:
Can we move this to a env config, or have some way of changing the target URL without having to update this code?

Reply:
@alexwybraniec in your mind would the env variable be the entire URL?